spec/rails_helper.rb

```diff
@@ -79,3 +79,10 @@
   config.filter_sensitive_data('<PROPUBLICA_API_KEY>') { ENV['PROPUBLICA_API_KEY'] }
```

Comment:
Why is this line in here? FE shouldn't need to filter any data since it's not calling external API?